extensions/wikia/TrackingOptIn/js/trackingOptIn.js

```diff
@@ -1,11 +1,25 @@
 define('wikia.trackingOptIn', [
+	'wikia.cookies',
 	'wikia.instantGlobals',
 	'wikia.lazyqueue',
 	'wikia.log',
 	'wikia.trackingOptInModal'
-], function (instantGlobals, lazyQueue, log, trackingOptInModal) {
+], function (cookies, instantGlobals, lazyQueue, log, trackingOptInModal) {
 	var optIn = false,
-		logGroup = 'wikia.trackingOptIn';
+		geoRequiresConsent = true,
+		instance = null,
```

Comment:
It seems that we don't need this variable in this scope

Reply:
Agree, removed/moved.